python/dist2wheel.py

```diff
@@ -73,33 +75,51 @@ def create_all(self):
         for artifact in self.artifacts:
             try:
                 self.path = artifact["path"]
-                self.platform_tag = self._fixup_platform_tag(artifact)
+                self._set_platform_props(artifact)
                 self.checksum = self._fix_checksum(artifact["extra"]["Checksum"])
                 self.size = os.path.getsize(self.path)
             except KeyError:
                 continue
 
             self.wheel_file = WHEEL_TEMPLATE.format(**self.__dict__).encode()
             self.metadata_file = METADATA_TEMPLATE.format(**self.__dict__).encode()
+            os.makedirs(self.wheel_dir, exist_ok=True)
             self._emit()
 
-    @staticmethod
-    def _fixup_platform_tag(artifact):
+    def _matrix(self):
+        return {
+            "darwin": {
+                "amd64": {
+                    "arch": "x86_64",
+                    "platform": "macosx_10_7_x86_64",
+                    "tags": [
+                        f"{self.py_tag}-{self.abi_tag}-macosx_10_7_x86_64",
+                    ],
+                },
+            },
+            "linux": {
+                "amd64": {
+                    "arch": "x86_64",
+                    "platform": "manylinux2014_x86_64.manylinux_2_17_x86_64.musllinux_1_1_x86_64",
+                    "tags": [
+                        f"{self.py_tag}-{self.abi_tag}-manylinux2014_x86_64",
+                        f"{self.py_tag}-{self.abi_tag}-manylinux_2_17_x86_64",
+                        f"{self.py_tag}-{self.abi_tag}-musllinux_1_1_x86_64",
```

Comment:
I suppose it bears asking: why the two `manylinux` tags?  Just the minimum supported version should be enough.  Or are the different `manylinux` tags incompatible with each other?  Because if so, yikes.

Reply:
To be honest, I was just copying `pip-binary-factory` 😄

Reply:
Since I know very little about this particular bit, I'm feeling inclined to leave it, unless you feel differently.